unbound.conf

```diff
@@ -24,7 +24,10 @@ server:
   do-daemonize: no
 
   prefetch: yes
+  prefetch-key: yes
+  target-fetch-policy: "-1 -1 -1 -1 -1"
```

Comment:
I don't quite understand the meaning of this setting. What is the benefit of setting it to `"-1 -1 -1 -1 -1"`?

Reply:
this seems to remove limit on rule of how deep to go down with queries while prefetching


[-1 == prefetch all](https://www.nlnetlabs.nl/documentation/unbound/doxygen/structiter__env.html#a52c186371468966fbab86295299494a8)


Recommended for enabling the latter: 

>  If  you  enable  it  consider  adding  more  numbers  after the tar-get-fetch-policy to increase the max depth that is checked to.